<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptWidget</class>
 <widget class="QWidget" name="ScriptWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Widget</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loadingPage">
      <layout class="QVBoxLayout" name="loadingLayout">
       <item>
        <spacer name="loadingTopSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="loadingLabel">
         <property name="text">
          <string>Loading script...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="loadingBottomSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="scriptPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="fileInfoLayout">
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="scriptTableView">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QPushButton" name="recordButton">
           <property name="text">
            <string>Record</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Record script actions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="runScriptButton">
           <property name="text">
            <string>Run</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Start or resume script execution</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pauseButton">
           <property name="text">
            <string>Pause</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Pause script execution at current position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetButton">
           <property name="text">
            <string>Reset</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Stop execution and reset to beginning</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save...</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Save script to file</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="errorPage">
      <layout class="QVBoxLayout" name="errorLayout">
       <item>
        <layout class="QVBoxLayout" name="errorFileInfoLayout">
        </layout>
       </item>
       <item>
        <widget class="QListView" name="errorListView">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>